src/context/FormDataContext.tsx

```diff
@@ -23,17 +23,17 @@ export const formInitialState: FormData = {
   } as AddressResponse,
 };
 
-interface FormDataType {
-  initState: FormData;
+interface FormDataContextProps {
+  initState?: FormData;
```

Comment:
Why is this optional now?

Reply:
Good question. So that argument is now set by default on line 36. This makes it easier to write and to test.

Before, both the provider and the initial object value needed to be imported:
```
import {
  FormDataContextProvider,
  formInitialState,
} from "../../src/context/FormDataContext";

....
<FormDataContextProvider initState={formInitialState}>
```

Now, only `FormDataContextProvider` needs to be imported.

For testing, the `initState` prop is always explicitly written but it's also `formInitialState` by default in `TestProviderWrapper`.

Reply:
So it's optional only in the sense that you don't need to explicitly provide a value because there will always be a default value.